src/Layers/TiledMapLayer.js

```diff
@@ -140,7 +140,10 @@ export var TiledMapLayer = TileLayer.extend({
             }
 
             this.fire('lodmap');
+          } else if (map.options.crs && map.options.crs.code && (map.options.crs.code.indexOf(sr) > -1) || (map.options.crs === CRS.EPSG4326 && (sr === 4326))) {
```

Comment:
Hmm, I think the second part is actually redundant here.
I looked at `CRS.EPSG4326` here https://github.com/Leaflet/Leaflet/blob/master/src/geo/crs/CRS.EPSG4326.js#L20 and it also contains `code` attribute, so the code would go in the `else if` block already after the first check that is `map.options.crs && map.options.crs.code && (map.options.crs.code.indexOf(sr) > -1)`

So at the end it looks to me that this `else if` block could be simplified to this
`} else if (map.options.crs && map.options.crs.code && map.options.crs.code.indexOf(sr) > -1) {`

Reply:
good catch!